config.ini

```diff
@@ -81,6 +81,7 @@ jwt_algorithm = HS256
 jwt_secret = vrwgLNWEffe45thh545yuby
```

Comment:
Hi @oddjobz do we still need to jwt_secret for the gateway? Not sure which of our service-to-service endpoint still expect this? We can take a look in the morning.